src/containerapp/azext_containerapp/_help.py

```diff
@@ -2131,3 +2131,42 @@
           az containerapp job registry set -n my-containerapp-job -g MyResourceGroup \\
               --server MyContainerappJobRegistry.azurecr.io --identity system-environment
 """
+
+# Maintenance Config Commands
+helps['containerapp env maintenance-config add'] = """
+    type: command
+    short-summary: Add Planned Maintenance to a Container App Environment
+    examples:
+    - name: Configure a Container App Environment to use a Planned Maintenance
+      text: |
+          az containerapp env maintenance-config add --env-name myEnv -g MyResourceGroup \\
+              --duration 10 --start-hour-utc 11 --weekday Sunday
+"""
+
+helps['containerapp env maintenance-config update'] = """
+    type: command
+    short-summary: Update Planned Maintenance in a Container App Environment
+    examples:
+    - name: Update the Planned Maintenance in a Container App Environment
+      text: |
+          az containerapp env maintenance-config update --env-name myEnv -g MyResourceGroup \\
+              --duration 8 --start-hour-utc 12 --weekday Thursday
+"""
+
+helps['containerapp env maintenance-config show'] = """
+    type: command
+    short-summary: Show Planned Maintenance in a Container App Environment
+    examples:
+    - name: Show Planned Maintenance
+      text: |
+          az containerapp env maintenance-config show --env-name myEnv -g MyResourceGroup
```

Comment:
Please remember to update the `help.py` after update the `--environment` to `--name`

Reply:
Hi @jijohn14  @p-bouchon 

Do we have any public doc that required the --environment, it should be `--name/-n` to consistent with other `az containerapp env` sub command. Thanks.